lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -200,6 +200,8 @@ export abstract class ClientApplication {
                 cpu: process.arch || "",
                 os: process.platform || "",
             },
+            persistencePlugin: this.config.cache!.cachePlugin,
```

Comment:
Just out of curiosity because I'm new to the non-null assertion operator, what does it do here? Is it just a way to tell the compiler that there's no way `this.config.cache` could be null or undefined at this point?

Reply:
Yep, that's right.